packages/core/minos-microservice-networks/minos/networks/discovery/clients/abc.py

```diff
@@ -28,6 +33,14 @@ def route(self) -> str:
         # noinspection HttpUrlsUsage
         return f"http://{self.host}:{self.port}"
 
+    @classmethod
+    def _from_config(cls, config: Config, **kwargs) -> DiscoveryClient:
+        discovery_config = config.get_discovery()
+
+        client_host = discovery_config.get("host")
+        client_port = discovery_config.get("port")
+        return cls(host=client_host, port=client_port, **kwargs)
```

Comment:
Could you add a test to check this code?

Reply:
Done!